scaler/size-helper.js

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2021 Google LLC
+ *
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License
+ */
+
+/*
+ * Helper class for various spanner size related calculations 
+ *
+ * The input parameters include information about a Spanner instance and
+ * they are sent by the poller function.
+ */
+
+class SizeHelper {
+  constructor(spannerParameters) {
+    this.units = (spannerParameters.units == 'NODES') ? "nodes" : "processing units";
```

Comment:
Good suggestion.
I've eliminated the helper altogether to avoid duplication and for consistency.
The current Size is now sent directly from the poller.